app/js/main.js

```diff
@@ -267,15 +269,15 @@ function startup() {
   btnFramePrevious.addEventListener("click", function() {
     if (curSelectedFrame > 1) {
         _displayFrame(curSelectedFrame - 1);
-    } else if (winMode === "capture") {
+    } else if (winMode === "capture" && totalFrames) {
```

Comment:
Why was this condition (and the one on line 280) made to be compound? If it is important, go ahead and make just these changes on `master` (it will be fine, this branch is already out-of-date and has conflicts).